src/helm/config/model_deployments.yaml

```diff
@@ -105,28 +105,69 @@ model_deployments:
 
 
   # Amazon
-  - name: amazon/titan-text-lite-v1
-    model_name: amazon/titan-text-lite-v1
+  # Titan on Amazon Bedrock
+
+  - name: amazon/amazon.titan-text-lite-v1
+    model_name: amazon/amazon.titan-text-lite-v1
     tokenizer_name: huggingface/gpt2
     max_sequence_length: 4000
     client_spec:
       class_name: "helm.clients.bedrock_client.BedrockTitanClient"
 
-  - name: amazon/titan-tg1-large
-    model_name: amazon/titan-tg1-large
-    tokenizer_name: huggingface/gpt2
+  - name: amazon/amazon.titan-text-express-v1
+    model_name: amazon/amazon.titan-text-express-v1
+    tokenizer_name: amazon/titan
     max_sequence_length: 8000
     client_spec:
       class_name: "helm.clients.bedrock_client.BedrockTitanClient"
+  
+  # Mistral on Amazon Bedrock
 
-  - name: amazon/titan-text-express-v1
-    model_name: amazon/titan-text-express-v1
-    tokenizer_name: huggingface/gpt2
+  - name: amazon/mistral.mistral-7b-instruct-v0:2
+    model_name: amazon/mistral.mistral-7b-instruct-v0:2
+    tokenizer_name: amazon/mistral
```

Comment:
There is a distinction between model deployments and models. A model (e.g. `mistralai/mistral-7b-instruct-v0.2`) can have multiple model deployments (e.g. `together/mistral-7b-instruct-v0.2` which is hosted on Together AI, `amazon/mistral.mistral-7b-instruct-v0.2` which is hosted on Amazon Bedrock).

You can specify in the run entry which model deployment you want to use for that model e.g.

```bash
helm-run -r mmlu:subject=anatomy,model=mistralai/mistral-7b-instruct-v0.2,model_deployment=amazon/mistral.mistral-7b-instruct-v0.2 --m 5 --suite demo
```

For model deployments, there's already an existing model with other model deployments, so you should reuse that model's name in the configuration. You also need to set the tokenizer and sequence length correctly.

Example change:
```yaml
    model_name: mistralai/mistral-7b-instruct-v0.2
    tokenizer_name: mistralai/Mistral-7B-Instruct-v0.2
```